### ISSUE TITLE
Possible bug with Abyssal AACI

### ISSUE THREAD

**Issue by Divinity123** [noro6/kc-web#6]
Hello,

I have been testing the anti-air formula recently and I used your tool as a reference. It works great so thank you for that.

However, I found a possible bug in the code. When I look at the minimum shot down (最低保証) from Abyssal fleet when AACI triggers, it is one more than what I expect.
It looks like you have added 1 to the fixed shot down on the Abyssal side even though the fixed shot down in the AACI parameters have already included everything. Is that intentional or is that a bug?
Thanks in advance!

Add 1: https://github.com/noro6/kc-web/blob/7f65699988f53448d1c95121e25dabe58c39f3c6/src/classes/aerialCombat/shootDownInfo.ts#L155
AACI parameters: https://github.com/noro6/kc-web/blob/4b3f5c677285834d46233c9c91cae3120e593ce5/src/classes/const.ts#L706
Vita code: https://github.com/rsghost/kancolle-kai-decompile-source/blob/master/Server_Controllers.BattleLogic/Exec_AirBattle.cs#L1055

**Comment by noro6** [comment]
I checked and it is possible that I was mistaken.
I have referred to this site (https://docs.google.com/document/d/1tqYyqzdc1RT_fYDKFMcUId0kOZHCdGpVsObm6yt-Yco/edit#heading=h.9md7m3uoezjq) in implementing these formulas.

The following statement is included in that text.
```
補足

仮設：
A'=対空CI不發の定數A
B'=対空CI不發の定數B

未方の最低擊墜上升が増加する=(A+B)-(A'+B') = A + B -1　= 固定ボーナス
相手の最低擊墜上升が増加する=(A+B)-(A'+B') = A + B =　固定ボーナス + 1

対空カットイン発動しない場合、定數Ａは強制的に０にされる。だが発動できればこれはしない。そのため敵の対空カットインの最低撃墜数はプレイヤー側のより１上がる
```

I had incorrectly interpreted that when Abyssal fleet triggers AACI, the fixed shot down is one more than when a friendly fleet triggers it.


To answer your question, the relevant code was intentionally added by me.
However, it is probably this code is actually unnecessary.

Thank you for pointing this out.
